src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -116,6 +116,7 @@ public OutsideRuntimeClient(ClientConfiguration cfg, bool secondary = false)
             services.AddFromExisting<IInternalGrainFactory, GrainFactory>();
             services.AddFromExisting<IGrainReferenceConverter, GrainFactory>();
             services.AddSingleton<MessageFactory>();
+            services.AddSingleton<Func<string, Logger>>(LogManager.GetLogger);
```

Comment:
I think it's OK for now... Clients are probably not using the Olreans logger anyway, and this opens it up for extensions to the runtime such as this.
Yes, I think we will revamp it... IMO we should use the Microsoft.Extensions.Logging package and get rid of our abstractions...